py3-seaborn.yaml

```diff
@@ -0,0 +1,39 @@
+# Generated from https://pypi.org/project/seaborn/
+package:
+    name: py3-seaborn
+    version: 0.13.0
+    epoch: 0
+    description: Statistical data visualization
+    copyright:
+        - license: BSD 3-Clause
+    dependencies:
+        runtime:
+            - py3-numpy
+            - py3-pandas
+            - py3-matplotlib
+            - python-3.11
+environment:
+    contents:
+        packages:
+            - ca-certificates-bundle
+            - wolfi-base
+            - build-base
+            - python-3.11
```

Comment:
Since 3.12 is coming out soon, we should think about whether this needs to be pinned to 3.11 or whether it can float to 3.12 when that becomes the latest py3.

If the latter, this should depend on `python3` and `py3-pip`, and when 3.12 comes out we'll bump this epoch to rebuild with 3.12.

If the former, we should understand why it needs to stay on 3.11 and document that somewhere, so we can check in some future version upgrade PR whether it can get on 3.12.

Reply:
Ah, I don't think this needs to be pinned to 3.11. I'll change the PR as you suggest.

Reply:
Fixed via commit 7b4550c